gitops/generate_build.go

```diff
@@ -232,10 +233,13 @@ func getParamsForComponentBuild(component appstudiov1alpha1.Component, isInitial
 	sourceCode := component.Spec.Source.GitSource.URL
 	outputImage := component.Spec.ContainerImage
 	var err error
+
+	if err = protectDefaultImageRepo(outputImage, component.Namespace); err != nil {
+		return []tektonapi.Param{}, err
+	}
+
 	if !isInitialBuild {
-		if outputImage, err = normalizeOutputImageURL(component.Spec.ContainerImage, component.Namespace); err != nil {
-			return []tektonapi.Param{}, err
-		}
+		outputImage = normalizeOutputImageURL(component.Spec.ContainerImage)
```

Comment:
for my education @Michkov - why do we not want to surface the error capturing the invalid user tag combination?

don't we want to invalidate the entire component interaction here?